README.md

```diff
@@ -26,11 +26,11 @@
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 portfolio-setup-and-mobile-first <a name="about-project"></a>
+# 📖 Capstone Project <a name="about-project"></a>
```

Comment:
- [ ] As the previous code reviewer mentioned, Please add the video link here, and also in the presentation video you have to show a piece of code that you are proud of. This will serve as documentation for your project.
![218090782-02808536-b33f-4a1f-b775-0ae0dfb1d858](https://user-images.githubusercontent.com/69077061/218248368-45540928-e5cc-4eab-a8be-8f86ef812928.png)